tests/unit/dom/select-files-test.js

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { triggerEvent, setupContext, teardownContext } from '@ember/test-helpers';
+import { buildInstrumentedElement } from '../../helpers/events';
+import hasEmberVersion from 'ember-test-helpers/has-ember-version';
+
+module('DOM Helper: selectFiles', function(hooks) {
+  if (!hasEmberVersion(2, 4)) {
+    return;
+  }
+
+  let context, element;
+
+  const textFile = new Blob(['Hello World'], { type: 'text/plain' });
```

Comment:
Do we need to guard this? It doesn't look like `Blob` is available in IE11 (and I want to make sure the test suite still passes there)...

Reply:
I could send in POJOs rather than skipping the test for IE entirely?